setup.cfg

```diff
@@ -0,0 +1,32 @@
+[metadata]
+name = poglink 
+description = "A bot that integrates the ARK Web API with Discord.",
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = "FM-17"
+author_email = "info@fm17.dev",
+url = "https://github.com/FM-17/poglink",
+license = MIT
+license_file = LICENSE
+project_urls =
+    Bug Tracker = https://github.com/fm-17/poglink/issues
+    DockerHub = https://hub.docker.com/r/fm17/poglink
+classifiers = 
+    Development Status :: 4 - Beta
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3.7
+    Framework :: AsyncIO
```

Comment:
A hasty selection of classifiers I found from [here](https://pypi.org/classifiers/), feel free to suggest additions,removals,modifications etc. Just thought it'd be "proper" to include.